### PR TITLE
Issue #8077: Hide 'Inspect Element with Firebug' menu item when e10s is enabled

### PR DESCRIPTION
**Pull request by SebastianZ**

### CODE REVIEW

File: extension/content/firebug/firefox/browserMenu.js [hunk 1]
```diff
@@ -436,15 +436,17 @@ var BrowserMenu =
     overlayFirefoxMenu: function(doc)
     {
         // Firefox page context menu
+      if (!this.browserOverlay.isMultiprocessEnabled() && !this.browserOverlay.auroraChannel) {
```
Comment (janodvarko) [inline extension/content/firebug/firefox/browserMenu.js]: The menu should be visible for users with e10s off, but why is the menu hidden for Aurora in this patch? 

nit: please fix the indentation (Firebug is using 4 spaces)

I've been testing with Aurora an the menu is always hidden no matter whether e10s is on or off. Btw. I removed the Aurora condition so, the code looked as follows:

```
if (this.browserOverlay.isMultiprocessEnabled()) {
}
``` 

The `isMultiprocessEnabled` function returns always false and need to be fixed.


My STR:
1) Launch Aurora
2) Go to Tools -> Options and uncheck "Enable multi-process Firefox Developer Edition", restart
3) Load e.g. google.com
4) Right click on the page and check that Firebug's inspect menu item is there. It isn't in my case -> BUG

Honza

**Review by janodvarko — CHANGES_REQUESTED** [review]
Thanks for the patch. Please see my inline comment.

**Comment by SebastianZ** [comment]
I've removed the Aurora channel check. There's really no reason to keep it. And I've fixed the indentation as requested.

> The `isMultiprocessEnabled` function returns always false and need to be fixed.

It works as expected for me, i.e. it returns `true` when e10s is enabled and otherwise `false`. (Tested on 50.0.1 through 54.0a1.) Doesn't it for you?

Sebastian

**Review by janodvarko — APPROVED** [review]
Works for me!

I was again testing `isMultiprocessEnabled()` and couldn't see any issues so, it all looks good.

Thanks for the patch!
Honza

**Pull request merged**